synfig-studio/src/gui/canvasview.cpp

```diff
@@ -1005,9 +1005,9 @@ CanvasView::create_time_bar()
 	timeslider->set_size_request(-1,header_height-header_height/3+1);
 	widget_kf_list->set_size_request(-1,header_height/3+1);
 
-	Gtk::Alignment *space = Gtk::manage(new Gtk::Alignment());
+	Gtk::Fixed *space = Gtk::manage(new Gtk::Fixed);
 	space->set_size_request(4);
-        space->show();
+    space->show();
```

Comment:
Wrong indentation :/

Reply:
Fixed Indentation :)

Reply:
No, it isn't ;) We use `tab` for source code identantion ;)

Reply:
I've fixed the indentation now ! just with 1 ``` tab ``` ! Kindly look into it :)